examples/graphql_example/migration/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-pub use sea_schema::migration::*;
+pub use entity::sea_orm::migration::prelude::*;
```

Comment:
why is this `entity::sea_orm` instead of `sea_orm`?

Reply:
we should make it `sea_orm_cli::migration` or `sea_orm_migration`

Reply:
`entity` is a pure logic crate it does not make sense for `migration` be imported via `entity`